lightblue/src/main/java/org/esbtools/eventhandler/lightblue/LightblueDocumentEventRepository.java

```diff
@@ -103,7 +103,28 @@ public void addNewDocumentEvents(Collection<? extends DocumentEvent> documentEve
                 .map(LightblueDocumentEventRepository::asEntity)
                 .collect(Collectors.toList());
 
-        lightblue.data(InsertRequests.documentEventsReturningOnlyIds(documentEventEntities));
+        int newEventsCount = documentEventEntities.size();
+        int maxEventsPerInsert = config.getMaxDocumentEventsPerInsert().orElse(newEventsCount);
+
+        int numberOfRequestsOfMaxEvents = newEventsCount / maxEventsPerInsert;
```

Comment:
Yeah I just thought this was clearer without type casting to / from double. Doesn't make much difference to me tho if you would rather see ceil(..) and casts